components/discordRp.ts

```diff
@@ -312,11 +312,7 @@ export function scenePathToRpAsset(
         // haven
         case "assembly:/_pro/scenes/missions/opulent/mission_stingray/scene_stingray_basic.entity":
         case "assembly:/_pro/scenes/missions/opulent/mission_stingray/scene_stingray_arcticthyme.entity":
-            return [
-                "havenlastresort",
-                "The Last Resort",
-                "Haven - The Maldives",
-            ]
+            return ["havenlastresort", "The Last Resort", "Haven"]
```

Comment:
The changes in this file are technically unrelated to the livesplit integration.

Reply:
They are - he's using this function to get mission names and such from scene paths.